main.py

```diff
@@ -1,7 +1,45 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    # Create
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+
+    # Update
+    drama_genre = Genre.objects.get(name="Dramma")
+    drama_genre.name = "Drama"
+    drama_genre.save()
+
+    george_klooney = Actor.objects.get(
+        first_name="George", last_name="Klooney"
+    )
+    george_klooney.last_name = "Clooney"
+    george_klooney.save()
+
+    kianu_reaves = Actor.objects.get(first_name="Kianu", last_name="Reaves")
+    kianu_reaves.first_name = "Keanu"
+    kianu_reaves.last_name = "Reeves"
```

Comment:
using the `update` method is generally considered more efficient, concise, and readable, as it reduces the number of database queries, improves performance, and ensures atomicity for simple update operations